### DISCUSSION TITLE
screamfest

### DISCUSSION THREAD

**Discussion by mememe0142** [iptv-org/community#797] [Q&A]

**Comment by BellezaEmporium** [comment]
https://vcnleomarkstudios.teleosmedia.com/stream/leomarkstudios/screamfest/playlist.m3u8

**Comment by freearhey** [comment]
https://vcnleomarkstudios.teleosmedia.com/stream/leomarkstudios/screamfest/playlist.m3u8 (offline)
<img width="639" alt="image" src="https://user-images.githubusercontent.com/7253922/232228753-fcab0336-08a7-4d63-a524-fd243423927d.png">

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/issues/19788
